server/mockEducationData.ts

```diff
@@ -0,0 +1,28 @@
+// TODO delete this file once the education view has been tested
+
+import type { EducationResponse } from 'educationAndWorkPlanApiClient'
+import EducationLevelValue from './enums/educationLevelValue'
+import QualificationLevelValue from './enums/qualificationLevelValue'
+
+const today = new Date()
+
+const mockEducationData: EducationResponse = {
```

Comment:
As mentioned above, lets get rid of this, as there are better ways to do it
(essentially it's what `server/testsupport/educationResponseTestDataBuilder.ts` should be doing 👍 , so if you wanted to use test/mock data, you would call `aValidEducationResponse()` rather than  `mockEducationData`)

Reply:
Yes this was another bad idea! Deleted.